src/components/BackEndRolePage/Brag.js

```diff
@@ -2,9 +2,18 @@ import React from 'react';
 
```

Comment:
Please name this files `RobotConfess`, `FoxBrag`, `WhaleDefault` etc. because quick navigation in VS Code gets really hard with such naming.

What about naming it `RobotSad`, `RobotConfident`, `RobotConfused`? I think it will be less confusing.

Reply:
I think probably leaving route as suffix makes more sense since in face those are routes wrappers.